docs(fix): Improve language and clarify that `cssFloat` is only needed in older browsers (#32069)
mdn/content: files/en-us/web/css/float/index.md

| `inline-grid`        | `grid`         |
| _other_              | _unchanged_    |

> > **Note:** When accessing a CSS property in JavaScript through the {{domxref("HTMLElement.style")}} object, single-word property names are used as is. So the CSS `float` property can be accessed as `float` in JavaScript. Keep in mind that `float` is a reserved keyword in JavaScript, but modern browsers support the direct use of `float`. In older browsers, however, you must use `cssFloat` to access the `float` property. (This is similar to how the "class" attribute is accessed as "className" and the "for" attribute of a \<label> element is accessed as "htmlFor".)

Reviewer: I tried making this a bit tighter:

```suggestion
> **Note:** When accessing a CSS property in JavaScript through the {{domxref("HTMLElement.style")}} object, single-word property names are used as is. The CSS `float` property is accessed as `float` in JavaScript. Although ' float` is a reserved keyword in JavaScript, modern browsers support the direct use of `float`. In older browsers, however, you must use `cssFloat` to access the `float` property. (This is similar to how the "class" attribute is accessed as "className" and the "for" attribute of a \<label> element is accessed as "htmlFor".)
```
